src/frontend/admin/src/services/api/models/OrderGroup.ts

```diff
@@ -1,11 +1,13 @@
 import { CourseProductRelation } from "@/services/api/models/Relations";
+import { Discount } from "@/services/api/models/Discount";
 
 export type OrderGroup = {
   id: string;
   nb_available_seats: number;
   nb_seats: number;
   is_active: boolean;
   can_edit: boolean;
+  discount?: Discount | null;
```

Comment:
When there is no discount the value will be always null so the property should not be optional.
```suggestion
  discount: Discount | null;
```